playbooks/edx_sandbox.yml

```diff
@@ -25,6 +25,7 @@
     SANDBOX_ENABLE_ECOMMERCE: False
     SANDBOX_ENABLE_DISCOVERY: False
     SANDBOX_ENABLE_CERTIFICATES: True
+    SANDBOX_ENABLE_RABBITMQ: False
```

Comment:
I'm thinking now, that may be better to rename it to SANDBOX_ENABLE_LOCAL_RABBITMQ, what do you think @UmanShahzad ?

Reply:
I think that variable is fine TBH; I understand that the IP used resolves to localhost, but this feels generic enough that can be re-used for local/non-local installs.

Reply:
If we want to upstream this change, this needs to default to `True` for backwards compatibility, and we need to set it to false in Ocim.  If we don't want to upstream this, it's fine to leave it as it is.